packages/frontend/src/components/send/components/RawTokenAmount.js

```diff
@@ -10,7 +10,7 @@ const RawTokenAmount = ({
     withSymbol = true,
     showFiatAmountForNonNearToken
 }) => {
-    if (decimals && symbol) {
+    if (decimals && symbol || decimals === 0 && symbol) {
```

Comment:
decimals in this component is being used to determine how many spots to the left to move the decimal over to.
ie if the amount is 1, and decimals is 5, the resulting calculation from the components is .00005. if i don't want to do this kind of formatting, the easiest way is to just say decimals 0.

Reply:
Could we do something like this instead?

`decimals = null,`

`if (decimals !== null && symbol)`